netbeans.apache.org/src/content/participate/report-issue.asciidoc

```diff
@@ -23,16 +23,10 @@
 :keywords: Apache NetBeans issue report
 :description: Apache NetBeans Reporting Issues
 
-If you have found a bug in the application, please help Apache NetBeans by reporting this problem
-to link:https://github.com/apache/netbeans/issues[our issue tracking on GitHub].
 
-Thank you for helping us make Apache NetBeans better!
-
-== Note
-
-Please make sure the issue can be replicated with the latest Apache NetBeans release,
+Please make sure the issue can be replicated with the link:https://netbeans.apache.org/download/[latest] Apache NetBeans release running on a supported JDK version,
```

Comment:
Possibly make supported JDK explicit similar to link to latest?  eg. "running on a supported JDK (currently Java 11+)."

Reply:
probably better to mention it directly yeah. It is not going to change that often :)

I am improving the exception reporter a little bit. Going to wait with merge till the actual PR is ready so that both can be looked at together.